packages/appsync-emulator-serverless/package.json

```diff
@@ -38,7 +38,7 @@
   },
   "devDependencies": {
     "apollo-boost": "^0.1.7",
-    "aws-appsync": "ConduitVC/aws-appsync",
+    "aws-appsync": "SET001/aws-appsync",
```

Comment:
what is this for?

Reply:
unless this will be fixed - https://github.com/ConduitVC/aws-appsync/issues/1

Reply:
try `yarn`, or try https://www.npmjs.com/package/aws-appsync